frontend/src/components/Header/Header.jsx

```diff
@@ -20,6 +20,57 @@ const Header = () => {
   const [showDrawer, setShowDrawer] = useState(false);
   const { pathname } = useLocation();
 
+  const smallHeader = (
+    <S.HeaderContent>
+      {inDev && <ThemeToggle />}
+      <div style={{ margin: "10px" }}>
+        <Button
+          type="primary"
+          size="large"
+          onClick={() => setShowDrawer(true)}
+          icon={<BarsOutlined style={{ color: "#fff", fontSize: "1.7rem" }} />}
+        />
+      </div>
+    </S.HeaderContent>
+  );
+
+  const largeHeader = (
+    <S.HeaderContent>
+      <Menu
+        theme="dark"
+        selectedKeys={[pathname.split("/")[1]]}
```

Comment:
you could use the optional `limit` parameter for [split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) here so that it doesn't split the whole string, to marginally improve performance to a perhaps negligible degree

Reply:
Thanks, will make this to another ticket

Reply:
I did not write this code.